Library/Formula/lua.rb

```diff
@@ -114,7 +117,7 @@ def install
   def pc_file; <<-EOS.undent
     V= 5.2
     R= 5.2.4
-    prefix=#{HOMEBREW_PREFIX}
+    prefix=#{prefix}
```

Comment:
Yeh, `prefix` sounds good for now but it's maybe worth considering this in future, yeh.